bundle/jsky.elevation.plot/src/main/java/jsky/plot/ElevationPlotModel.java

```diff
@@ -426,6 +429,14 @@ public XYDataset getConstraintsDataset() {
         return new double[]{startAlt, endAlt};
     }
 
+    // Altitude in degrees at the target's declination at the given time, the given hour angle, and
+    // the observer's latitude, or None if the target's coordinates are unknown.
+    private static Option<Double> altit(TargetDesc target, Date when, double ha, double lat) {
+        return target.getCoordinates(new Some(when.getTime())).map(coords ->
+            ImprovedSkyCalcMethods.altit(coords.getDecDeg(), ha, lat)
+        );
+    }
+
     // Returns true if the given elevation value meets the constraints, otherwise false.
```

Comment:
So, I changed it to compute `start/endAlt` at the start and end dates. I'm not sure if that is correct but it doesn't seem any worse than what's there now.